### ISSUE TITLE
Imagine That....

### ISSUE THREAD

**Issue by agautsc** [codeforgreenville/UpstateEvents#8]
This site was first focused on a single thing. May I suggest a site that is focused on getting food and drink sponsors for events.  I would call it pizzabeerorcoffee.com and it list all the groups that have regular events. They post their event info and what they are looking for from a sponsor. Then sponsors can reach out to organizer directly and make it happen. 

That way, a single site could be marketed to all the places that like to be sponsors for these type of events.

In fact, please do this or I might just slap together something quickly that does it.

**Comment by allella** [comment]
@agautsc

Landon and I had a talk about how to get some value out of what we have already before tackling the larger API.

We're looking to solve some smaller pains with the smaller data, but also get people open to the idea of leveraging open API data. For instance, IFTTT style things like allowing people to be notified when a new tech group is added, or when notable things are changed.

I wrote an email around lifeingreenville.com to see if their site, which has a few out of data lists (albeit small and curated) could get value from community lists like the few we know about. Maybe there are more.

For instance, for new people, and API of tech groups
http://greenville.orangecoat.net/organizations/all (human)
https://github.com/codeforgreenville/UpstateEvents/issues/17 (robots)

Or a map of free Wi-Fi
https://joinopenworks.com/wifi/

Or a list/map of startups (Jeremy Boeh)
http://864startups.com/

It's still kind of new, but the map layers we talked about last week with Jimmy Thigpen is another potential one.

Landon and I weren't coming up with a lot of great ideas for using the data we have, or uses for an incremental data set.  Seems like a job for ImagineThat, but not sure enough folks are involved to set one up yet for just this data set.  Maybe something to keep in the mind about ideas for open data sets for Greenville, in general.
